src/views/asset-edit-view/__snapshots__/asset-edit-view.test.tsx.snap

```diff
@@ -16,7 +16,7 @@ exports[`renders the whole 'Edit property address' view 1`] = `
   <span
     class="govuk-caption-m lbh-caption"
   >
-    New Addresses are suggested from the Local Gazetteer to bring some standardisation.
+    Addresses are suggested from the Local Gazetteer to bring some standardisation.
```

Comment:
Is there a reason for 'New' being removed?

Reply:
I read it and thought it shouldn't be there. They're not "new" addresses, they're just suggestions for existing addresses. If you disagree I'm happy to leave it as is though